.github/workflows/publish-version.yml

```diff
@@ -124,6 +124,45 @@ jobs:
         with:
           token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
 
+      - name: Download binaries
+        uses: actions/download-artifact@v2
+        with:
+          path: release
+
+      - name: Move binaries to root directory
+        run: mv release/*/stacks-wallet* .
+
+      - name: Download release-notes.txt from create-version workflow
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: create-version.yml
+          name: release-notes
+
+      - name: Kill first line of release notes
+        run: sed -i '1d' release-notes.txt
+
+      - name: Append with opening code block
+        run: echo "\`\`\`" >> release-notes.txt
+
+      - name: Create shasums file
+        run: shasum -a 256 stacks-wallet* >> shasums.txt
+
+      - name: Import GPG key
+        id: import_gpg_key
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+      - name: Gpg armor
+        run: gpg --armor --output shasums.txt.asc --clearsign shasums.txt
+
+      - name: Concat release notes and shasums
+        run: cat release-notes.txt shasums.txt.asc > release-body.txt
+
+      - name: Append with closing code block
+        run: echo "\`\`\`" >> release-body.txt
```

Comment:
Same here regarding simplification. Ideally any `run` steps performed in a sequence can be boiled down into a single step to resemble a small script. This can help make the workflow file a bit shorter and easier to comprehend.

If some of the `run` titles help organize the logic of a command, you can add regular bash comments to the run step script to keep it short.